common/client_config.go

```diff
@@ -0,0 +1,40 @@
+package common
+
+import (
+	"fmt"
+	"time"
+
+	clients_v2 "github.com/Layr-Labs/eigenda/api/clients/v2"
+)
+
+// ClientConfigV2 contains all non-sensitive configuration to construct V2 clients
+type ClientConfigV2 struct {
+	// Enabled is true if using eigenDA v2, or false if using eigenDA v1
+	Enabled                  bool
+	DisperserClientCfg       clients_v2.DisperserClientConfig
+	PayloadDisperserCfg      clients_v2.PayloadDisperserConfig
+	RelayPayloadRetrieverCfg clients_v2.RelayPayloadRetrieverConfig
+	// hex address of the EigenDAServiceManager contract
+	ServiceManagerAddress string
+	PutRetries            uint
+	// interval at which the ethereum node is polled, while waiting to reach a given block number
+	BlockNumberPollIntervalDuration time.Duration
```

Comment:
What if a user passes 0 here by mistake?

Reply:
Cert verifier [interprets](https://github.com/Layr-Labs/eigenda/blob/e36cbb4637efbde1f889664dd67c2b3628ef5571/api/clients/v2/verification/cert_verifier.go#L79) interval <= 0 to mean "don't wait for block to advance", and any call made with a behind eth node will fail immediately